doc/source/report/plots/hist_kolmogorov.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import stats
+
+from mousestyles.distribution.kolmogorov_test import get_travel_distances
+from mousestyles.distribution.kolmogorov_test import perform_kstest
+
+
+def plot_histogram(strain=0, mouse=0, day=0, distribution=stats.pareto,
+                   n_hist_bins=50):
+    """ Plot the histogram of distances travelled by a mouse in a day within 20
+    milliseconds.
+    We then fit a distribution as specified by the argument distribution,
+    using Maximum Likelihood, and plot the histgoram of that fitted
+    distribution.
+    Finally, we fit a distribution by minimizing CDF distance, and plot the
+    histogram of that fitted distribution.
+
+    In addition to the histograms (PDF), the CDF are also provided.
+    Parameters:
+    -----------
+    strain: int
+        Denote the strain of mouse to plot
+    mouse: int
+        Denote the the mouse id (what twin) within the strain
+    day: int
+        Denote the day to plot
+    distribution: scipy.stats.rv_continuous object
+        {stats.pareto, stats.expon, stats.gamma}
+        A distribution to fit the data to. Currently supporting three
+        distributions.
+    """
+    dist_name = distribution.__class__.__name__.split("_")[0]
+    distances = get_travel_distances(strain, mouse, day)
+    mle_params = distribution.fit(distances)
+    opt_params = perform_kstest(strain, mouse, day, verbose=False)
```

Comment:
This is syntactically invalid.  Your function takes 3 arguments and you provide 4.  Also please include this plot in the report.